package.json

```diff
@@ -27,7 +27,7 @@
   "homepage": "https://github.com/davewasmer/devcert#readme",
   "devDependencies": {
     "standard-version": "^4.3.0",
-    "typescript": "^2.7.0"
+    "typescript": "^2.9.2"
```

Comment:
Did you find that you needed to update typescript and tslib for your changes to work? If not, I'd like to save dep updates for separate PRs.

Reply:
@zetlen yes, at the time I needed to update typescript for `flatMap` to work